artifactory/commands/generic/upload.go

```diff
@@ -271,3 +279,30 @@ func createDeleteSpecForSync(deletePattern string, syncDeletesProp string) *spec
 		Recursive(true).
 		BuildSpec()
 }
+
+func recordUploadJobSummary(summary *rtServicesUtils.OperationSummary) (err error) {
```

Comment:
recordUploadJobSummary --> recordSummaryData

Let's a clear description for this function, allowing random readers who know nothing about this functinality to understand it.

Let's add a unit test for this function.